src/pages/manageUser.tsx

```diff
@@ -6,24 +7,139 @@ import InputAdornment from '@mui/material/InputAdornment';
 import ManageCentersModal from '@/components/ManageCentersModal';
 import ManageUsersModal from '@/components/ManageUsersModal';
 import MoreVertIcon from '@mui/icons-material/MoreVert';
-import React from 'react';
 import SearchIcon from '@mui/icons-material/Search';
 import Tab from '@mui/material/Tab';
 import Tabs from '@mui/material/Tabs';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import { useRouter } from 'next/router';
 import { useTheme } from '@mui/material/styles';
 import { useTranslation } from 'next-i18next';
+import { useEffect } from 'react';
+import { getFacilitorList } from '@/services/ManageUser';
+import { cohortList } from '@/services/CohortServices';
+
+interface Cohort {
+  cohortId: string;
+  parentId: string;
+  name: string;
+}
+
+type CohortsData = {
+  [userId: string]: Cohort[];
+};
 
 const manageUser = () => {
   const { t } = useTranslation();
   const theme = useTheme<any>();
   const router = useRouter();
 
   const [value, setValue] = React.useState(1);
+  const [users, setUsers] = useState<
+    { name: string; district: string; userId: string }[]
+  >([]);
+  const [loading, setLoading] = React.useState(false);
+  const [cohortsData, setCohortsData] = useState<CohortsData>({});
+  const [open, setOpen] = React.useState(false);
+  const [selectedUser, setSelectedUser] = useState(null);
+  const [selectedUserName, setSelectedUserName] = useState(null);
+  const [centers, setCenters] = useState<string[]>([]);
   const handleChange = (event: React.SyntheticEvent, newValue: number) => {
     setValue(newValue);
   };
+
+  useEffect(() => {
+    const getFacilitator = async () => {
+      try {
+        let state, district;
+        if (typeof window !== 'undefined' && window.localStorage) {
+          state = localStorage.getItem('state');
+          district = localStorage.getItem('district');
+        }
+        if (state && district) {
```

Comment:
what if state and district is not available??

Reply:
API call will not happen